x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/e5_multilingual_callout/deploy_model.tsx

```diff
@@ -0,0 +1,119 @@
+  isDismissable,
+}: Pick<
+  E5MultilingualCallOutState,
+  'dismiss' | 'ingestionMethod' | 'isCreateButtonDisabled' | 'isDismissable'
+>) => {
+  const { createE5MultilingualModel } = useActions(E5MultilingualCalloutLogic);
+
+  return (
+    <EuiCallOut color="primary">
+      <EuiFlexGroup direction="column" gutterSize="s">
+        <EuiFlexItem>
+          <EuiFlexGroup direction="row" gutterSize="s" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <EuiBadge color="primary">
+                <FormattedMessage
+                  id="xpack.enterpriseSearch.content.index.pipelines.e5MultilingualCallOut.titleBadge"
+                  defaultMessage="New"
+                />
+              </EuiBadge>
+            </EuiFlexItem>
+            <EuiFlexItem grow>
+              <EuiText color="primary" size="xs">
+                <h3>
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.content.index.pipelines.e5MultilingualCallOut.title',
+                    { defaultMessage: 'Improve your results with E5' }
+                  )}
+                </h3>
+              </EuiText>
+            </EuiFlexItem>
+            {isDismissable && (
+              <EuiFlexItem grow={false}>
+                <E5MultilingualDismissButton dismiss={dismiss} />
+              </EuiFlexItem>
+            )}
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiFlexGroup direction="column">
+            <EuiFlexItem>
+              <EuiText size="s">
+                <FormattedHTMLMessage
+                  id="xpack.enterpriseSearch.content.index.pipelines.e5MultilingualCallOut.body"
+                  defaultMessage="E5 produces dense vector embeddings that can be searched in multiple languages."
+                  tagName="p"
+                />
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiFlexGroup
+                direction="row"
+                gutterSize="m"
+                alignItems="center"
+                justifyContent="flexStart"
+              >
+                <EuiFlexItem grow={false}>
+                  <EuiButton
+                    color="primary"
+                    data-telemetry-id={`entSearchContent-${ingestionMethod}-pipelines-e5MultilingualCallOut-deployModel`}
+                    disabled={isCreateButtonDisabled}
+                    iconType="launch"
+                    onClick={() => createE5MultilingualModel()}
+                  >
+                    {i18n.translate(
+                      'xpack.enterpriseSearch.content.indices.pipelines.e5MultilingualCallOut.deployButton.label',
+                      {
+                        defaultMessage: 'Deploy',
+                      }
+                    )}
+                  </EuiButton>
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EuiLink target="_blank" href={docLinks.elser}>
```

Comment:
@demjened Is there any doc link for e5?

Reply:
I didn't see ES docs, but I did find a blog post: https://www.elastic.co/search-labs/blog/articles/multilingual-vector-search-e5-embedding-model

Reply:
updated